.gitignore

```diff
@@ -1,3 +1,7 @@
+# Editor files
+*.vs
+*.vscode
```

Comment:
These have been added already in another PR :]

Reply:
I think it is fine, it will just create a collision if one of these PRs gets merge before...